init/start.sh

```diff
@@ -83,4 +83,5 @@ exec sudo -u ${CE_USER} -H --preserve-env=NODE_ENV -- \
     --env amazon \
     --port 10240 \
     --static out/dist \
```

Comment:
Oh hang on! Don't we need to remove this (the PR in compiler-explorer removes the `--static` command-line option entirely..